.ruby-version

```diff
@@ -1 +1 @@
-ruby-2.5.7
\ No newline at end of file
+ruby-2.6.3
```

Comment:
Why is it 2.6.3? The newest version of 2.6 series is [2.6.5](https://www.ruby-lang.org/en/news/2019/10/01/ruby-2-6-5-released/)

Reply:
Nem figyeltem mi a legújabb, de simán lehet 2.6.5